numpy/core/memmap.py

```diff
@@ -17,6 +17,14 @@
     "write":"w+"
     }
 
+mmap_defaults = {
+    'r':{'access':mmap.ACCESS_READ},
```

Comment:
Need to import `mmap`.

Reply:
Yeah thanks.  I just saw that `mmap` has lazy import and I respect that now.